ESSArch_Core/test_util.py

```diff
@@ -173,3 +177,155 @@ def test_get_schema_from_file_with_bad_input(self):
 
         with self.assertRaises(etree.XMLSyntaxError):
             getSchemas(filename=fp)
+
+
+class NestedLookupTest(TestCase):
```

Comment:
Should also include tests for multiple occurrences of the key, and a test for when the key doesn't exist anywhere in the document